docs/beast_development.rst

```diff
@@ -42,97 +42,85 @@ project is:
 Development Install
 ===================
 
-It is much easier to perform development if any changes you make to the code are
-immediately reflected in how the ``beast`` runs (in contrast to needed to perform a
-new install each team). This can be achieved by using a development install.
+It is much easier to develop the code if any changes you make are immediately
+reflected in how the ``beast`` runs (in contrast to performing a new install each
+team). This can be achieved by using a development install. First, create a fork

```

Comment:
team -->time

Reply:
I just ended up simplifying this part -- thanks though!.